course_content/notebooks/dask_intro.ipynb

```diff
@@ -393,6 +393,281 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(bag_cubelist)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "type(bag_cubelist)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is worth noting that part of the beauty of Python is that we do not necessarily need to create four objects (`fp`, `files`,  `bag_o_cubes`, `bag_cubelist`) to get a `CubeList` object but can chain the functions together into one big function call:\n",
+    "        \n",
+    "        iris.cube.CubeList(db.from_sequence(files).map(iris.load_cube).compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load multiple files into single cube using `iris.load`\n",
+    "If the data you are loading from multiple files represents the same phenomenon then when the data is stored as multiple cubes can be merged into a single cube. For instance from our previous example we have 120 cubes each representing the northward sea ice velocity in a single month and collectively representing that data for the decade 1890-1899. We can merge this dataset into a single cube using Iris as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "merged_cube = iris.load(files).merge_cube()\n",
+    "\n",
+    "print(merge_cube)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compare this to the ouput of printing a single cube and it is clear that the decade's worth of data has been merged into a single cube with 120 time steps:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "single_cube = iris.load_cube(files[0])\n",
+    "\n",
+    "print(single_cube)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load multiple files into single cube using `dask.bag` and `dask.delayed`\n",
+    "We can merge the `CubeList` we generate using a `dask.bag` in exactly the same way:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bag_merged_cube = bag_cubelist.merge_cube()\n",
+    "\n",
+    "print(bag_merged_cube)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "However the `merge_cube` step in this does not leverage Dask. We can utilise Dask in the merge step by decorating a call to `merge_cube` with the `@delayed` decorator. This allows us to integrate it with our task graph:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@delayed\n",
+    "def del_merge(cubes):\n",
```

Comment:
The contents of this code cell are excellent. I wonder whether we should rename `del_*` to `delayed_*` for clarity though, as `del` in Python already has a clearly defined meaning as the delete operation (which is what I immediately thought was going on here).

Reply:
#111 addresses :white_check_mark: